ansible/roles/dropbear_initramfs/templates/etc/initramfs-tools/scripts/local-bottom/debops_dropbear_initramfs.j2

```diff
@@ -1,9 +1,9 @@
+{# Copyright (C) 2015-2017 Robin Schneider <ypid@riseup.net>
+ # Copyright (C) 2017 DebOps <https://debops.org/>
+ # SPDX-License-Identifier: GPL-3.0-only
+ #}
 #!/bin/sh
```

Comment:
I think that in the case of shell scripts, the Copyright and SPDX license strings should be put under the shebang (`#!/bin/sh`). This should allow the `shellcheck` linter to properly detect such scripts. In that case you cannot put this in the Jinja comment tags because shell linting will break... So basically reverse this change. :-)

Reply:
I see what you did there with the other shell script templates. They are valid shell scripts even before being templated. This works in some cases. I looked at the loop in this file and I don’t see an easy way to make it valid before.

You already disabled `shellcheck` for this file in 8cb601b82c7395237ae00e58f74d5a628d3c2938. I would leave it like this.

Reply:
But I agree with you that for consistency, this commit should be undone. I removed it.